data/sql/updates/pending_db_world/winter-reveler.sql

```diff
@@ -0,0 +1,50 @@
+DELETE FROM creature_addon WHERE guid IN (SELECT guid FROM creature WHERE id=15760);
```

Comment:
This is wrong, I'll fix it when I add the sniffed Dalaran NPC auras.

Reply:
I know that integers aren't supposed to have apostrophes (') around them, but since `auras` is a text datatype, I don't know if that impacts things. If they also shouldn't have surrounding apostrophes, I can remove those.